conda_gitenv/deploy.py

```diff
@@ -136,6 +152,9 @@ def deploy_repo(repo, target):
 def configure_parser(parser):
     parser.add_argument('repo_uri', help='Repo to deploy.')
     parser.add_argument('target', help='Location to deploy the environments to.')
+    parser.add_argument('env_labels', nargs='*',  default=None, 
+                        help='Specific labelled environments to deploy, in the '
+                             'form "{environment}/{label}". Optional argument.',)
```

Comment:
How about matching a pattern, something like:

```
>>> env_labels = [']
>>> fnmatch.fnmatch('default/current', 'default/*')
```

The default behaviour is currently ```'*'```.